nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -4,6 +4,12 @@
 
 ## Highlights {#sec-release-24.11-highlights}
 
+- **This will be the latest version of Nixpkgs to support macOS 10.12 (Sierra).**
```

Comment:
```suggestion
- **This will be the last version of Nixpkgs to support macOS 10.12 (Sierra).**
```
Typo? I don't think *latest* can be used to denote the last iteration of something that has ended.

Reply:
Yes, typo. I’ll try to open a new PR to tighten up the wording here today.